vaadin-custom-field-flow-integration-tests/src/main/java/com/vaadin/flow/component/customfield/examples/CustomFieldDemoPage.java

```diff
@@ -0,0 +1,162 @@
+package com.vaadin.flow.component.customfield.examples;
+
+import com.vaadin.flow.component.customfield.CustomField;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Input;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.timepicker.TimePicker;
+import com.vaadin.flow.demo.DemoView;
+import com.vaadin.flow.router.Route;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.function.Function;
+
+@Route("vaadin-custom-field-test-demo")
+public class CustomFieldDemoPage extends DemoView {
```

Comment:
Do we need it? It's not used in tests.

Reply:
Done. I removed the CustomFieldDemoPage.java as it has nothing to do with the test.